tests/test_files/json/fs_as_array/ser-ref/casWithSofaDataArray/data.json

```diff
@@ -0,0 +1,21 @@
+{
+  "%TYPES" : { },
+  "%VIEWS" : {
+    "_InitialView" : {
+      "%SOFA" : 0,
+      "%INDEX" : [ ]
+    }
+  },
+  "%FEATURE_STRUCTURES" : [ {
+    "%ID" : 1,
+    "%TYPE" : "uima.cas.ByteArray",
+    "%ELEMENTS" : "VGhpcyBpcyBhIHRlc3Q="
```

Comment:
Why is the type of a byte array a string? Is that how UIMA rolls or is it temporary?

Reply:
Not sure what you mean. If you mean that "elements" is a base64-encoded string, then I'd say it is simply what makes sense - it is also the default how Jackson serializes byte arrays into JSON.

Reply:
I would have thought it is an array of ints < 255

Reply:
That is way less space-efficient.

Reply:
How does UIMAj serialize it with XMI?

Reply:
https://github.com/apache/uima-uimaj/blob/main/uimaj-core/src/test/resources/CasSerializationDeserialization_XMI_1_0_PRETTY_Test/ser-ref/casWithSofaDataArray/data.xmi

```
<?xml version="1.0" encoding="UTF-8"?><xmi:XMI xmlns:tcas="http:///uima/tcas.ecore" xmlns:xmi="http://www.omg.org/XMI" xmlns:cas="http:///uima/cas.ecore" xmi:version="2.0">
    <cas:NULL xmi:id="0"/>
    <cas:Sofa xmi:id="2" sofaNum="1" sofaID="_InitialView" mimeType="text/plain" sofaArray="1"/>
    <cas:ByteArray xmi:id="1" elements="5468697320697320612074657374"/>
</xmi:XMI>
```

Reply:
But hex strings are old-skool, no?